docs/current_docs/manuals/administrator/tutorials/index.mdx

```diff
@@ -0,0 +1,12 @@
+---
+slug: /manuals/administrator/tutorials
+displayed_sidebar: "current"
+---
+
+# Tutorials
+
+Learn more about Dagger using the step-by-step walkthroughs in this section.
+
+- [Configure a Custom Registry Mirror for Dagger](./272381-custom-registry.mdx)
```

Comment:
```suggestion
```

This can be a top-level section in the Administrator manual.

Reply:
Done